x-pack/packetbeat/magefile.go

```diff
@@ -143,10 +149,41 @@ func TestPackages() error {
 	return devtools.TestPackages()
 }
 
-func TestSystem(ctx context.Context) error {
-	mg.Deps(devtools.BuildSystemTestBinary)
+func SystemTest(ctx context.Context) error {
+	mg.SerialDeps(getNpcapInstaller, devtools.BuildSystemTestBinary)
 
 	args := devtools.DefaultGoTestIntegrationArgs()
 	args.Packages = []string{"./tests/system/..."}
 	return devtools.GoTest(ctx, args)
 }
+
+// getNpcapInstaller gets the installer from the Google Cloud Storage service.
+//
+// On Windows platforms, if getNpcapInstaller is invoked with the environment variables
+// CI or NPCAP_LOCAL set to "true" and the OEM Npcap installer is not available it is
+// obtained from the cloud storage. This behaviour requires access to the private image.
```

Comment:
> This behaviour requires access to the private image.

This make me think it's accessing the docker image to get the installer. Consider rewording.

Reply:
s/image/store/